src/bvh/mod.rs

```diff
@@ -0,0 +1,9 @@
+//! This module defines a [`BVH`].
+//!
+//! [`BVH`]: struct.BVH.html
+//!
+
+pub use self::bvh::*;
```

Comment:
The [Rust Book](https://doc.rust-lang.org/book/crates-and-modules.html) does it this way:
```
pub use self::greetings::hello;
pub use self::farewells::goodbye;

mod greetings;
mod farewells;
```